vars/withSubscriptionLogin.groovy

```diff
@@ -32,4 +32,4 @@ def call(String subscription, boolean alwaysExecute, Closure body) {
 
 def call(String subscription, Closure body) {
```

Comment:
@timja the pipeline is complaining about this being a duplicate method of the same signature. Can this block be removed or should it be renamed?

Reply:
remove it as it's no longer needed